### DISCUSSION TITLE
Raspberry Pi OS 11

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#121] [Q&A] [answered]
I just installed Qt Creator 4.11.0 based on Qt 5.12.8 on the raspberry but no predefined selection kit

do you have a simple solution please?

**Comment by textbrowser** [comment]
So you downloaded the ARM Debian file (https://github.com/textbrowser/biblioteq/releases/download/2021.10.30/BiblioteQ-2021.10.30_armhf.deb) and you ran "sudo apt install ..." on it and that did not work?

**Comment by textbrowser** [comment]
This is my Pi development machine's information:
pi@snoopy:~ $ lsb_release -a
No LSB modules are available.
Distributor ID: Raspbian
Description:    Raspbian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye
pi@snoopy:~ $ qmake --version
QMake version 3.1
Using Qt version 5.15.2 in /usr/lib/arm-linux-gnueabihf

My test machine (a separate Pi) does not include Qt. The information I posted yesterday regarding the installation (sudo apt install ...) results show that the Debian file installs correctly.

**Comment by meteos77** [comment]
your package works very well on pi400

I just wanted to compile the updated version of your developments and so it is :
- either compilation on the pi but problem of the missing kit hence my question (because the help forums are not up to date with the new version of raspios 11)
- or cross-compiling from my main machine which I don't know how to do yet (my to do list)
- or you make a package ... but I know it's not the priority (so no request from me)

in conclusion : I manage, I close the subject

**Comment by textbrowser** [comment]
I build BQ directly on a Pi 4. The Qt 5 development packages are required. I do not document the required packages because I do not wish to maintain yet another list. A developer will have the knowledge to do that; or an administrator. :)

I only cross-compile the Android release.

**Comment by textbrowser** [comment]
> qmake -o Makefile biblioteq.arm.pro
> make -j $(nproc)

See also Documentation/README.RASPBIAN for details on creating the package for Pi.

**Comment by textbrowser** [comment, marked as answer]
And! The control file (RASPBIAN/control) documents the required packages. :)

**Comment by meteos77** [comment]
I redid a test on pi400 to make a demo on a TV for the volunteers of the library.
os : raspios-bullseye-arm64
The 2022-03-30 version works perfectly (7200 documents base)
